<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SendButtonTemplatesEditor</class>
 <widget class="QDialog" name="SendButtonTemplatesEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>464</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Template editor</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Templates</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="QLineEdit" name="txtTemplate">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line1">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QTreeWidget" name="lstTemplates">
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <property name="dragEnabled">
           <bool>true</bool>
          </property>
          <property name="dragDropMode">
           <enum>QAbstractItemView::InternalMove</enum>
          </property>
          <property name="defaultDropAction">
           <enum>Qt::MoveAction</enum>
          </property>
          <attribute name="headerVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="headerVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string notr="true">1</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout">
          <item>
           <widget class="QToolButton" name="btnUp">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>26</width>
              <height>51</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">margin:0;padding:0;</string>
            </property>
            <property name="arrowType">
             <enum>Qt::UpArrow</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="btnDown">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>26</width>
              <height>51</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">margin:0;padding:0;</string>
            </property>
            <property name="arrowType">
             <enum>Qt::DownArrow</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="btnAdd">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Edit</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnRemove">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Remove</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnOk">
       <property name="text">
        <string>Ok</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>txtTemplate</tabstop>
  <tabstop>lstTemplates</tabstop>
  <tabstop>btnUp</tabstop>
  <tabstop>btnDown</tabstop>
  <tabstop>btnAdd</tabstop>
  <tabstop>btnEdit</tabstop>
  <tabstop>btnRemove</tabstop>
  <tabstop>btnOk</tabstop>
  <tabstop>btnCancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btnOk</sender>
   <signal>clicked()</signal>
   <receiver>SendButtonTemplatesEditor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>265</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel">
     <x>-2</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>SendButtonTemplatesEditor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>341</x>
     <y>283</y>
    </hint>
    <hint type="destinationlabel">
     <x>81</x>
     <y>272</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnRemove</sender>
   <signal>clicked()</signal>
   <receiver>SendButtonTemplatesEditor</receiver>
   <slot>removeTemplate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>227</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnEdit</sender>
   <signal>clicked()</signal>
   <receiver>SendButtonTemplatesEditor</receiver>
   <slot>editTemplate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>231</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel">
     <x>231</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnUp</sender>
   <signal>clicked()</signal>
   <receiver>SendButtonTemplatesEditor</receiver>
   <slot>upTemplate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>439</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>231</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnDown</sender>
   <signal>clicked()</signal>
   <receiver>SendButtonTemplatesEditor</receiver>
   <slot>downTemplate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>439</x>
     <y>196</y>
    </hint>
    <hint type="destinationlabel">
     <x>231</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lstTemplates</sender>
   <signal>customContextMenuRequested(QPoint)</signal>
   <receiver>SendButtonTemplatesEditor</receiver>
   <slot>contextMenu(QPoint)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>219</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>231</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lstTemplates</sender>
   <signal>currentItemChanged(QTreeWidgetItem*,QTreeWidgetItem*)</signal>
   <receiver>SendButtonTemplatesEditor</receiver>
   <slot>selectionChanged(QTreeWidgetItem*,QTreeWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>219</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>231</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>removeTemplate()</slot>
  <slot>editTemplate()</slot>
  <slot>upTemplate()</slot>
  <slot>downTemplate()</slot>
  <slot>selectionChanged(QTreeWidgetItem*,QTreeWidgetItem*)</slot>
  <slot>contextMenu(QPoint)</slot>
 </slots>
</ui>
